### PR TITLE
Skateboards have adjustable speed, can fit in backpacks

### PR DESCRIPTION
**Pull request by Mickyan**
:cl: Mickyan
balance: Skateboards can fit in backpacks
tweak: Skateboards are slower by default, speed can be adjusted by alt-clicking
/:cl:

Skateboards are fun but without even considering the bumping hazards they're almost impossible to use on station just because of how unreasonably fast they are. 
They're so fast, they couldn't be any faster. Literally! They have no move delay, making them as fast as a vehicle can possibly be.
But if you think you can handle it, you can simply adjust it back to sanic speed. Crashing penalties are unaffected.

I see no reason why they shouldn't fit in backpacks, maybe they used to be strong weapons a while ago but nowadays you can find stronger stuff just laying around that fit in backpacks just fine. 
Plenty of items bigger than a skateboard that already fit in backpacks, too. (i.e. instruments)

### CODE REVIEW

**Comment by evsey9** [comment]
Not RADICAL. Its either SKATE or DIE.

**Comment by Anonmare** [comment]
It's skate, or be skated

**Comment by Repukan** [comment]
If you can't handle the heat, get out of the kitchen

**Comment by TheWiznard** [comment]
Isn't that why you can make a scooter?

**Comment by PKPenguin321** [comment]
👍 wish there was a better way than construction. And yeah, I made them not fit in bags since they had higher than average damage. Also so you can't just plop down a skateboard wherever you want on a dime and immediately start riding it away, but that probably isn't that much of an issue realistically

**Comment by Mickyan** [comment]
Escaping using a skateboard sounds like a terrible idea and I have nothing but respect for anyone that manages to pull it off without crashing and failing miserably

seriously though, so long as meth is a thing (or hell, you can get nuka cola out of soda vendors) skateboards are just going to be a fun gimmick

**Comment by Bawhoppen** [comment]
Couldn't we just decrease their force if there's a concern about them being a powerful weapon?

**Comment by GuppyLaxx** [comment]
who cares, 12 force is fucking nothing

File: code/modules/vehicles/scooter.dm [hunk 1]
```diff
@@ -92,6 +93,17 @@
...
+	var/datum/component/riding/R = src.GetComponent(/datum/component/riding)
+	if (!adjusted_speed)
+		R.vehicle_move_delay = 0
+		to_chat(user, "<span class='notice'>You adjust the wheels on [src] to make it go faster.</span>")
+		adjusted_speed = TRUE
+	else
+		R.vehicle_move_delay = 1
+		to_chat(user, "<span class='notice'>You adjust the wheels on [src] to make it go slower.</span>")
+		adjusted_speed = FALSE
+
```
Comment (yoyobatty) [inline code/modules/vehicles/scooter.dm]: This could be shrunk down by a couple lines.

**Pull request merged**